scripts/pre-commit/validate_changelog.py

```diff
@@ -0,0 +1,119 @@
+import subprocess
+import logging
+import re
+import sys
+import typing as tp
+from pathlib import Path
+
+logger = logging.getLogger("validate_changelog_update")
```

Comment:
May be you also want to update the logger name, although it doesn't matter much.